stencil.config.ts

```diff
@@ -79,6 +79,9 @@ export const config: Config = {
       "^/assets/(.*)$": "<rootDir>/src/tests/iconPathDataStub.js",
     },
   },
+  hydratedFlag: {
+    selector: "attribute",
```

Comment:
👍. would we want it to be more customized to something like `calcite-hydrated`?

Reply:
That's probably a good idea. The style it adds for this _is _ global, just like the hydrated css class, so prefixing it is probably smart.

Reply:
Will wait for others to chime in on different naming ideas before swapping everything.

Reply:
Yep - name-spacing it makes sense. Great addition, it's kind of odd that isn't the default OOTB.